src/NodaTime.Web/Markdown/root/versions.md

```diff
@@ -708,3 +711,16 @@ Bug fixes:
 ## 1.0.0-beta1, released 2012-04-12 with tzdb 2012c
 
 - Initial beta release
+
+# <a name="tzdb-updates"></a>Update policy for time zone data
+
+New time zone data is released by IANA as required. The latest data
+is always available via the Noda Time web site as described in the
+[user guide](/userguide/tzdb). Additionally, NuGet packages are
```

Comment:
Maybe:

```
in the [user guide]().

Additionally, released versions of the NodaTime NuGet package are also updated...

(next para)
```

Reply:
I think I'd rather not have two single-line paragraphs.